doc/plugins.md

```diff
@@ -274,48 +276,56 @@ The list of plugins:
 
```

Comment:
Maybe you could add something like the following to address plugins which aren’t in this list:

> [find more plugins on npmjs](https://www.npmjs.com/search?q=keywords:remark-plugin)

Reply:
It’s a bit higher already: https://github.com/remarkjs/remark/blob/5969b6f53205c08b075a1e0d62e0ac07e9595d7b/doc/plugins.md?plain=1#L19-L22